iohk/NixOps.hs

```diff
@@ -408,7 +408,7 @@ readTopology file = do
     Left err -> errorT $ format ("Failed to parse topology file: "%fp%": "%w) file err
 
 data SimpleTopo
-  =  SimpleTopo (Map.Map NodeName SimpleNode)
+  =  SimpleTopo { fromSimpleTopo :: (Map.Map NodeName SimpleNode) }
```

Comment:
Shouldn't this be newtype?

Reply:
It started as a `newtype`, right -- but for some reason I had to turn this into a `data`.  Don't remember why : -/

Besides, there is no _real_ difference (except performance, which I think we can agree is not an issue here) between these options.